bindings/python/setup.py

```diff
@@ -89,20 +94,31 @@ def write_b2_python_config(target, config):
     paths = sysconfig.get_paths()
     includes = [paths["include"], paths["platinclude"]]
 
-    # Note that on debian, the extension suffix is overwritten, but it's
-    # necessary everywhere else, or else b2 will just build "libtorrent.so".
-    # python.jam appends SHLIB_SUFFIX on its own.
-    target = target.name.split("libtorrent")[1]
-    if os.name == "nt":
-        suffix = sysconfig.get_config_var("EXT_SUFFIX")
-    else:
-        suffix = sysconfig.get_config_var("SHLIB_SUFFIX")
-    if suffix != None and target.endswith(suffix):
-        target = target[:-len(suffix)]
-
-    using = f'using python : {sysconfig.get_python_version()} : {escape(sys.executable)} : {" ".join(escape(path) for path in includes)} : : <libtorrent-python>on : "{target}" ;\n'
-    print(using)
-    write(using)
+    write("using python")
+    write(f" : {sysconfig.get_python_version()}")
+    write(f" : {b2_escape(sys.executable)}")
+    write(" : ")
+    write(" ".join(b2_escape(path) for path in includes))
+    write(" :")  # libraries
+    write(" : <libtorrent-python>on")
+
+    # Note that sysconfig and distutils.sysconfig disagree here, especially on
+    # windows.
+    ext_suffix = distutils.sysconfig.get_config_var("EXT_SUFFIX")
+
+    # Note that debian currently disregards ext_suffix, but it overrides it
+    # using the same logic we use. Hopefully any system-specific patches would
+    # do the same thing.
+
+    # python.jam appends the platform-specific final suffix on its own. I can't
+    # find a consistent value from sysconfig or distutils.sysconfig for this.
+    for plat_suffix in (".pyd", ".dll", ".so", ".sl"):
```

Comment:
what's ".sl"?

Reply:
Something used on hpux apparently. This list is from here:
https://github.com/boostorg/build/blob/e0870c43214a1d5e4c476eacf9f67dae90ec0b56/src/tools/python.jam#L1094-L1110